core/input/util.js

```diff
@@ -175,6 +175,10 @@ export function getKeysym(evt) {
         return DOMKeyTable[key][location];
     }
 
+    if(key === "Dead" ){
+        return getDeadKeysym(evt);
+    }
+
```

Comment:
seems to me that `key==="Dead"` just works in Firefox and not in Chrome .

In firefox I got all accents working perfectly . 

I' like use electron app , because I can use ctrl + key , how we catch dead keys in chrome as it here ?